File: Server/Server/src/Core/Application/Identity/Users/UserDetailsDto.cs
namespace Server.Application.Identity.Users;

public class UserDetailsDto
{
    public Guid Id { get; set; }

    public string? UserName { get; set; }

    public string? FirstName { get; set; }

    public string? LastName { get; set; }

    public string? Email { get; set; }

    public bool IsActive { get; set; } = true;

    public bool EmailConfirmed { get; set; }

    public string? PhoneNumber { get; set; }

    public string? ImageUrl { get; set; }
    public DateTime BirthDate { get; set; } = Convert.ToDateTime("01/01/1900");
    public string? CustomerName { get; set; }
    public string? CustomerId { get; set; }
    public string? EntCode { get; set; }
}

File: client/src/app/models/user/userDetailsDto.ts
export interface UserDetailsDto {
  id: string;  
  userName?: string;
  firstName?: string;
  lastName?: string;
  email?: string;
  isActive: boolean;
  emailConfirmed: boolean;
  phoneNumber?: string;
  imageUrl?: string;
  birthDate?:Date;
  customerName?:string;
  customerId?:string;
  entCode?:string;
}

            


